unstructured/file_utils/filetype.py

```diff
@@ -363,6 +383,29 @@ def document_to_element_list(
     return elements
 
 
+PROGRAMMING_LANGUAGES = [
+    "javascript",
+    "python",
+    "java",
+    "c++",
+    "cpp",
+    "csharp",
+    "c#",
+    "php",
+    "ruby",
+    "swift",
+    "typescript",
+    "go",
```

Comment:
This one seems susceptible to misclassification, I can find a few mime types with the letters `go` in them. There are mime types related to google applications that I think would be triggered, as well as one `application/x-seelogo`.

Reply:
Yeah good call, I'll do an explicit check against `text/x-go` for that one.